crates/unleash-edge/src/edge_builder.rs

```diff
@@ -265,6 +271,13 @@ pub async fn build_edge(
     ))
 }
 
+pub fn hosting_type() -> Hosting {
```

Comment:
The into/map/from_str dance really isn't necessary here. We literally care about a single case here - if the env var is set and it's hosted